### ISSUE TITLE
trace udevadmin LTC issue

### ISSUE THREAD

**Issue by bybai** [xcat2/xcat-core#3011]
trace udevadmin LTC issue: https://bugzilla.linux.ibm.com/show_bug.cgi?id=153609
If this issue is fixed, update xcat getinstdisk code.

**Comment by bybai** [comment]
Based on LTC comments, I uploaded unsuccessful and successful log.

**Comment by bybai** [comment]
Hi @whowutwut , I got comments from LTC as following:
```
------- Comment From gduarte@redhat.com 2017-06-14 14:46:47 EDT-------
This looks like a duplicate of RHBZ 1438976 (LTC 153199).

Please update the firmware to skiboot-5.6.0 and retest.
```

**Comment by bybai** [comment]
"udevadm info --attribute-walk --name=/dev/sda" in %pre section of kickst can work well when firmware is skiboot-5.6.0-rc2 on p9euh02.

**Comment by bybai** [comment]
Fixed in #3204, will verify in autotest env.

**Comment by robin2008** [comment]
Yuan have already verified in autotest env, and it works.
